### PR TITLE
Add integration test for NodeResourceLimits plugin

### LINKED ISSUE #86926: Add integration tests for NodeResourceLimits scoring function
**What would you like to be added**:
An integration test for NodeResourceLimits plugin.

**Why is this needed**:
No end-to-end test coverage current exist for this alpha feature.

/sig scheduling
/help

### PR DESCRIPTION
**Pull request by notpad**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

An integration test for NodeResourceLimits plugin.

**Which issue(s) this PR fixes**:

Fixes #86926

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/sig-scheduling
/assign @ahg-g

### CODE REVIEW

File: test/integration/scheduler/plugins_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package scheduler
```
Comment (oomichi) [inline test/integration/scheduler/plugins_test.go]: pull-kubernetes-verify job is failed like:
```
Boilerplate header is wrong for: /home/prow/go/src/k8s.io/kubernetes/test/integration/scheduler/plugins_test.go
```
I think that requires the apache license header lines in the file like the other go files.

**Review by oomichi — COMMENTED** [review]
/cc @oomichi

**Comment by ahg-g** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**